diskquota.c

```diff
@@ -735,10 +736,48 @@ disk_quota_launcher_main(Datum main_arg)
 		 */
 		if (got_sigusr1)
 		{
+			elog(DEBUG1, "[diskquota] got sigusr1");
 			got_sigusr1 = false;
 			sigusr1     = true;
 		}
 
+		/*
+		 * Try to starts a bgworker for the curDB
+		 *
+		 */
+
+		/*
+		 * When db list is empty, curDB is NULL.
+		 * When curDB->in_use is false means dbEtnry has been romoved
+		 * When curDB->dbid doesn't equtal curDBId, it means the slot has
+		 * been used by another db
+		 *
+		 * For the above conditions, we just skip this loop and try to fetch
+		 * next db to run.
+		 */
+		if (curDB == NULL || !curDB->in_use || curDB->dbid != curDBId)
+		{
+			advance_one_db = true;
+			continue;
+		}
+
+		/*
+		 * Try to start a worker to run the db if has exceeded the next_run_time
```

Comment:
```suggestion
		 * Try to start a worker to run the db if has exceeded the next_run_time.
```
Add a `.` to avoid confusion.